openreview/api/client.py

```diff
@@ -2813,3 +2835,13 @@ def post(self, client):
         client.post_group(self)
 
 
+    def transform_to_anon_ids(self, elements):
+        if self.anonids:
+            for index, element in enumerate(elements):
+                if element in self.members and self.anon_members:
```

Comment:
Is it necessary to initialize self.members and self.anon_members to empty arrays in the constructor?

Reply:
I think you are right, I need to initialize them